Benchmarks/Package.swift

```diff
@@ -0,0 +1,27 @@
+// swift-tools-version: 5.7
+
+import PackageDescription
+
+let package = Package(
+    name: "benchmarks",
+    platforms: [
+        .macOS("13.3"),
+    ],
+    dependencies: [
+        .package(path: "../"),
+        .package(url: "https://github.com/ordo-one/package-benchmark.git", from: "1.11.1"),
+    ],
+    targets: [
+        .executableTarget(
+            name: "Benchmarks",
+            dependencies: [
+                .product(name: "FoundationEssentials", package: "swift-foundation"),
+                .product(name: "Benchmark", package: "package-benchmark"),
+            ],
+            path: "Benchmarks/Predicates",
```

Comment:
It looks like this is a great executable target for all of our benchmarks, but it specifies this `Benchmarks/Predicates` folder specifically. What if we renamed this `Benchmarks` subfolder to `Sources` or something similar (to avoid confusion with the `Benchmarks` folder in the root directory of this repo) and moved `Predicates.swift` up a level instead of putting it in a subfolder. Then we could just set path here to `"Sources"` (if that's even needed) and it'll let us easily add new files for non-predicate benchmarks in the future. Thoughts?

Reply:
Short comment, this wouldn't work:

> The source for all benchmarks must reside in a directory named Benchmarks in the root of your swift package.
The benchmark plugin uses this directory combined with the executable target information to automatically discover and run your benchmarks.

https://swiftpackageindex.com/ordo-one/package-benchmark/1.11.2/documentation/benchmark/gettingstarted

Reply:
So most more complex benchmark suites would set up multiple executable targets (eg. One per subsystem, similar to unit test suites) - this makes manual benchmark driving easier (using --target) of a given suite too. 

Depending on the ambitions, it's easy to see several test suites (eg. Datetime, json, etc) but perhaps Predicates should be grouped with something else (the "essentials" grouping is perhaps a bit coarse though)

Reply:
Ah, good to know! Thanks for the pointer - sorry I'm not as familiar with this benchmarking library just yet 🙂. That makes sense. In that case, maybe we just change the name of this target to `PredicateBenchmarks` instead of `Benchmarks` and we can make it more broad in the future if we decide to add things to it (or just add on more targets)?